docker/docker-compose.yml

```diff
@@ -159,8 +162,7 @@ services:
     healthcheck:
       test: ["CMD", "curl", "-f", "http://localhost:8080/actuator/health"]
       start_period: 240s
-    image: quay.io/ukhomeofficedigital/hocs-casework
-    #image: hocs-casework-test
+    image: quay.io/ukhomeofficedigital/hocs-casework:${CASEWORK_TAG-branch-main}
```

Comment:
Does the following need the colon like the fronend above?

Reply:
It works with both syntax's!! I've change them all to ${X_TAG:-branch-main} for consistancy